CONTRIBUTING.md

```diff
@@ -229,6 +229,25 @@ Run script to download dashboards to `./dashboards` folder
 node scripts/download_dashboards.mjs
 ```
 
+## Contributing to Documentation
+
+When submitting PRs for documentation updates, build the docs locally and ensure functionality before submission. To build docs locally, ensure you have Python installed, then follow these steps:
+
+```sh
+pip install -r docs/requirements.txt
+yarn build:docs
+cd docs
+mkdocs serve --watch pages
+```
+
+Your locally served docs will then be accessible at http://localhost:8000.
+
+If you run into build issues due to circular dependencies, branch switching or other incompatibilities, try cleaning your modules and rebuild your dependencies with:
+
+```
+yarn clean && yarn clean:nm && yarn && yarn build
+```
+
```

Comment:
Might be worth to highlight how to extend the word list in case there is a unknown new word that cases spellcheck in CI to fail. 

- extend [.wordlist.txt](https://github.com/ChainSafe/lodestar/blob/unstable/.wordlist.txt)
- make sure it sorted `./scripts/wordlist_sort.sh`